app/controllers/auth.js

```diff
@@ -9,9 +9,6 @@ module.exports = {
       const token = await user.generateAuthToken()
       res.send({ user: user, token: token })
     } catch (error) {
-      if (process.env.NODE_ENV !== 'production') {
-        console.log(error.name, '-', error.message)
-      }
```

Comment:
if you are removing these logs how would we have a log when something goes wrong? and what was the data as this particular time when the error occoured?

Reply:
yes, you could check the logs in root directory of the folder /app/logs/app.log. There you could find all those logs where error occured and why occured etc.

Reply:
If you want to check it, just check this PR locally once

Reply:
perfect 👍

Reply:
See the log: The bold part is the data 
```
[2020-07-26 14:52:09] info: ::1 - - [26/Jul/2020:09:22:09 +0000] "GET /user/overview" 304 - "http://localhost:3000/dashboard" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.38 Safari/537.36" **{}**
```